### DISCUSSION TITLE
Color brand for login page and backend

### DISCUSSION THREAD

**Discussion by sohamakl** [typemill/typemill#444] [Ideas]
It would be nice if it were possible to adapt the color of the login page to the theme or even better if this were possible for the entire backend. Is there already a possibility for this, or can it be added as a suggestion for improvement?

**Comment by trendschau** [comment]
There is only a dark-mode for the backend right now. Brand colors for the backend are not supported and I don't know. To be honest, I don't know a lot of cms who allow branding of the backend. I think this is one of the spots where the product itself needs a bit branding. So if I create a plugin or feature for that, I would probably do that with a Business-license.
